src/brod_client.erl

```diff
@@ -843,20 +869,20 @@ do_start_consumer(TopicName, ConsumerConfig, State) ->
       end,
   do_on_valid_topic(TopicName, State, F).
 
+%% @private
 do_on_valid_topic(TopicName, State, F) ->
```

Comment:
rename to `ensure_partition_workers`
was planing to use it as a more generic function, but so far, it is only to start workers.